peps/pep-0790.rst

```diff
@@ -0,0 +1,469 @@
+PEP: 790
+Title: New environment markers for modern Python environments
+Author: Klaus Zimmermann <klaus_zimmermann@gmx.de>,
+        Ralf Gommers <ralf.gommers@gmail.com>
+Sponsor: Lysandros Nikolaou <lisandrosnik@gmail.com>
+Discussions-To: https://discuss.python.org/t/environment-marker-for-free-threading/60007
```

Comment:
Use the short links -- but I would instead put this existing discussion in Post-History, and leave Discussions-To empty until this PR is merged (then you will open a new discussion thread).
```suggestion
Discussions-To: https://discuss.python.org/t/60007
```